packages/terra-content-container/tests/nightwatch/DefaultContentContainer.jsx

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 
 import ContentContainer from '../../lib/ContentContainer';
```

Comment:
Lets move all these test examples to the new dir structure to drop reference to nightwatch all together
```
/component
    /examples
        /test-examples
            ...examples
```

Reply:
Updated: https://github.com/cerner/terra-core/pull/1468/commits/9790d7179668344164cbceea5bcf83e822169d6a